lighthouse/messages/broker.py

```diff
@@ -11,37 +11,47 @@
 class Broker:
     """Controls the connection, exchange and publishing to RabbitMQ."""
 
-    def connect(self) -> None:
-        self.__create_connection()
-        self.__open_channel()
+    def _connect(self) -> None:
```

Comment:
if _connect is private, how can we use the broker? I don't see in the public methods how we can create a new connection.